gh_pages/_source/setup/PC-Setup---ROS-Kinetic.md

```diff
@@ -23,8 +23,12 @@ The Virtual Box is limited both in hardware capability(due to VM limitations) an
 On most new systems, Virtual Box and VMs work out of the box.  The following is a list of issues others have encountered and solutions:
   * Virtualization must be enabled - Older systems do not have virtualization enabled (by default).  Virtualization must be enabled in the BIOS.  See <http://www.sysprobs.com/disable-enable-virtualization-technology-bios> for more information.
 
-## PC Configuration (**NOT Recommended**)
-An installation [shell script](https://github.com/AustinDeric/ros-industrial-env) is provided to run in Ubuntu Linux 16.04 (Xenial Xerus) LTS.  This script loads all the programs needed and configures the environment.
+## Direct Linux PC Configuration (**NOT Recommended**)
+An installation [shell script](https://github.com/AustinDeric/ros-industrial-env) is provided to run in Ubuntu Linux 16.04 (Xenial Xerus) LTS.  This script installs ROS and any other packages needed for the environment used for this training.
```

Comment:
Not sure how I feel about pointing to a script in a personal repo.  Could we copy this file into this repository? with @AustinDeric consent of course.

Reply:
:+1: to that

Reply:
@jdlangs ping

Reply:
@jdlangs could you copy that script into this repo?